lib/renaissance/helpers/compare.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Renaissance.Helpers.Compare do
+  alias Renaissance.Helpers.Adapt
+
+  def amount?(%Money{}), do: true
+
+  def amount?(p) do
+    if is_integer(p), do: true, else: false
+  end
+
+  def money_max(m1, m2) do
+    cond do
+      amount?(m1) && amount?(m2) == true ->
+        if Adapt.money_value(m1) >= Adapt.money_value(m2) do
```

Comment:
Can you use the `Money` comparison functions here? https://hexdocs.pm/money/Money.html#compare/2